### DISCUSSION TITLE
Get the hash form?

### DISCUSSION THREAD

**Discussion by usman-anwar-mindoula** [ElMassimo/oj_serializers#6] [Q&A] [answered]
Hello,

Is there a way to get the hash form from the serializer, instead of the JSON string?

**Comment by ElMassimo** [comment]
Hi Usman!

Not in the current version, though I was intending to add one for testing purposes.

Would love to hear about your use case, as most of the time you wouldn't need the hash form (building a hash is less performant than writing straight to JSON).

**Comment by usman-anwar-mindoula** [comment]
We are migrating from AMS, and some areas of our app use the hash form for various reasons (not rendering to the client). Since there isn't a `serializable_hash` method, we will probably end up calling `Oj.load` on the JSON string. Thanks.

**Reply by ElMassimo** [reply]
Gotcha. If you run into issues with that, I wouldn't mind adding `one_as_hash` and `many_as_hash` to the library, and pull requests are welcome.

**Comment by ElMassimo** [comment, marked as answer]
Rendering to a `Hash` is now the default behavior (`default_format :hash`), and can be also done explicitly with `one_as_hash` and `many_as_hash`.
